src/Passes/Source/Rules/Notation/Call.hpp

```diff
@@ -0,0 +1,69 @@
+#pragma once
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/// @defgroup shorthandNotation Shorthand Notation
+
+#include "Rules/Notation/Call.ipp"
```

Comment:
(I looked at your links about .ipp files. Thank you for those links)

FYI: The Call.ipp file provides the _definition_ (of the template function) first, and then the remaining part of this header (Call.hpp) provides the _declaration_. So the order of declaration and definition seems to me _reversed_ in all the translation units that include this Call.hpp. Such an order makes it questionable the break-up into separate .hpp (with declaration) and .ipp (with definition). I personally would consider merging these two files. But up to you.